src/generators/secrets/index.ts

```diff
@@ -29,9 +31,6 @@ class SecretsGenerator extends BaseGenerator {
         }
     }
 
-    console.log(answers)
-
-
     this.fs.copyTpl(
```

Comment:
Line 23 breaks the fundamental way that the scaffolder is expected to work.  You can either prompt the user for input or they can supply it form the command line.